app/models/event.rb

```diff
@@ -84,6 +85,14 @@ def fetch_github_stats
     stats
   end
 
+  def upcoming?
```

Comment:
@alexshook - To keep things consistent, would you mind adjusting your `Event#upcoming?` and `Event#past?` methods to refer to the `end_date` attribute instead? In the existing pattern, current events should be upcoming, not past.